src/main.tsx

```
@@ -1,14 +1,37 @@
+	useEffect(() => {
+		console.log(getItemById(treeStructure, "2"));
+	}, [treeStructure]);
+
+	return (
+		<SortableTree
+			items={treeStructure.items}
+			onItemsChange={setTreeStructure}
+			isRemovable
+			allowNestedItemAddition
+			showDropIndicator
+		></SortableTree>
```

Biome

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)